### PR TITLE
README.rst: Add PyPI badge; moved badges to top

### PR DESCRIPTION
**Pull request by msabramo**
Preview at https://github.com/msabramo/Kotti/tree/badges

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -19,6 +21,8 @@ thus hopefully keeping the learning curve flat for the developer.
 
 .. |build status| image:: https://secure.travis-ci.org/Kotti/Kotti.png?branch=stable
 .. _build status: http://travis-ci.org/Kotti/Kotti
+.. |pypi| image:: https://pypip.in/version/zope.interface/badge.svg?style=flat
```
Comment (disko) [inline README.rst]: I Think this needs to be https://pypip.in/version/Kotti/badge.svg?style=flat instead?

Comment (msabramo) [inline README.rst]: Oops :smile:

Comment (msabramo) [inline README.rst]: Fixed.

**Comment by disko** [comment]
Thanks!

**Pull request merged**